src/client/__test__/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '../App';
+
+describe('App component', () => {
+  const wrapper = shallow(<App />);
```

Comment:
is it fine with `shallow` here? cause what I understood from the docs and all is it will do a partial rendering right !...
Wouldnt it be better to use `render`?

Reply:
Yes, it would partially render but since we have tests for all components it is fine. A little issue with render is that the components might need certain props to render properly. Hence I prefer using shallow over render unless absolutely necessary.

Reply:
ohhkay great then